elastalert/elastalert.py

```diff
@@ -152,6 +153,10 @@ def __init__(self, args):
         self.disabled_rules = []
         self.replace_dots_in_field_names = self.conf.get('replace_dots_in_field_names', False)
         self.string_multi_field_name = self.conf.get('string_multi_field_name', False)
+        self.statsd_prefix = socket.gethostname()
```

Comment:
this needs to be configured, either in the config files or as a command line parameter

Reply:
not sure i understand why 
(this gets the container name if hostname set in ansible docker_container)

Reply:
done